imports/plugins/custom/completed-orders/client/containers/customMainDropdown.js

```diff
@@ -10,6 +10,9 @@ import { getRawComponent,
 import { Reaction, Logger }  from "/client/api";
 const MainDropdownComponent = getRawComponent("MainDropdown");
 
+import onboard from "/imports/plugins/custom/vendor-onboarding/client/onboard";
+import { dashboardSteps } from "/imports/plugins/custom/vendor-onboarding/client/tourSteps";
```

Comment:
Shouldn't these imports come before the `MainDropdownComponent` variable assignment?